src/components/CategoryPage.svelte

```diff
@@ -34,20 +34,12 @@
 
 <CategoryHeading {variableData} {variable} {category} location={selectedGeographyDisplayName} />
 
-<div class="tw-p-6 tw-bg-onspale tw-mb-6">
-  <a class="tw-hyperlink" href={buildHyperlink({ selectedGeography: $selectedGeographyStore })}>Home</a>
-  <span class="tw-mx-1">&gt;</span>
-  <a
-    class="tw-hyperlink"
-    href={buildHyperlink({
-      topic: topic.slug,
-      selectedGeography: $selectedGeographyStore,
-    })}>{topic.name}</a
-  >
-  <span class="tw-hidden xl:tw-inline">
-    <span class="tw-mx-1">&gt;</span>
-    {variable.name}
-  </span>
-</div>
-
+<NavigationComponent
+  {search}
+  {topicSlug}
+  currentURL={$page.url.pathname}
+  onClick={() => {
+    console.log($page.url.pathname);
```

Comment:
Just noting this here but I know this will replaced by @James-Nitron so all good here.